x-pack/plugins/ml/public/application/components/rule_editor/condition_expression.js

```diff
@@ -129,7 +126,7 @@ export class ConditionExpression extends Component {
 
   renderOperatorValuePopover() {
     return (
-      <div style={POPOVER_STYLE}>
+      <div>
```

Comment:
Looks like you need to apply this same edit to the scope expression popover too:

<img width="505" alt="image" src="https://user-images.githubusercontent.com/7405507/191565578-52ab3c42-2c8d-4763-beec-7ee607f8f702.png">

Reply:
Fixed in https://github.com/elastic/kibana/pull/141215/commits/b19671d6e4b5e5f6ddc1246668b6e9030337e9c0